Packs/IllusiveNetworks/Integrations/IllusiveNetworks/IllusiveNetworks.py

```diff
@@ -628,6 +647,149 @@ def get_event_incident_id_command(client: Client, args: dict) -> Tuple:
+
+    readable_output = tableToMarkdown('Illusive get incident\'s events', events)
+
+    outputs = {
+        'Illusive.Incident(val.incidentId == obj.incidentId)': {
+            'eventsNumber': len(events),
+            'incidentId': int(incident_id),
+            'Event': events
+        }
+    }
+    return (
+        readable_output,
+        outputs,
+        events  # raw response - the original response
+    )
+
+
+def get_forensics_analyzers_command(client: Client, args: dict) -> Tuple:
+    event_id = args.get("event_id", 0)
+    try:
+        analyzers = client.get_forensics_analyzers(event_id)
+        incident = client.get_event_incident_id(event_id)
+    except DemistoException as e:
+        if "404" in e.args[0]:
+            raise DemistoException("Event id {} doesn't not exist".format(event_id))
+        elif "429" in e.args[0]:
+            raise DemistoException(
+                "The allowed amount of API calls per minute in Illusive Attack Management has exceeded. In case this"
+                " message repeats, please contact Illusive Networks support")
+        else:
+            raise DemistoException("{}".format(e.args[0]))
+
+    outputs = {
+        'Illusive.Event(val.eventId == obj.eventId)': {
+            'eventId': int(event_id),
+            'incidentId': int(incident),
+            'ForensicsAnalyzers': analyzers
+        }
+    }
+
+    readable_output = tableToMarkdown('Illusive Forensics Analyzers', analyzers)
+
+    return (
+        readable_output,
+        outputs,
+        incident  # raw response - the original response
+    )
+
+
+def get_forensics_triggering_process_info_command(client: Client, args: dict) -> Tuple:
+    event_id = args.get("event_id")
+    try:
+        processes = client.get_forensics_triggering_process_info(event_id)
+    except DemistoException as e:
+        if "404" in e.args[0]:
+            raise DemistoException("failed to get forensics for Event id {}".format(event_id))
+        elif "429" in e.args[0]:
+            raise DemistoException(
+                "The allowed amount of API calls per minute in Illusive Attack Management has exceeded. In case this"
+                " message repeats, please contact Illusive Networks support")
+        else:
+            raise DemistoException("{}".format(e.args[0]))
+
+    readable_output = tableToMarkdown('Illusive Triggering Processes Info', processes.get('processes'))
+    outputs = {
+        'Illusive.Event(val.eventId == obj.eventId)': {
+            'eventId': event_id,
+            'ForensicsTriggeringProcess': processes.get('processes')
+        }
+    }
+
+    return (
+        readable_output,
+        outputs,
+        readable_output  # raw response - the original response
+    )
+
+
+def get_forensics_artifacts_command(client: Client, args: dict):
+    event_id = args.get("event_id")
+    artifact_type = args.get("artifact_type", "DESKTOP_SCREENSHOT")
+    try:
+        client.get_event_incident_id(event_id)
```

Comment:
you are not using this

Reply:
I know, it's for catching error in case of event that doesn't exist. 
(there is a know limitation in the next command)

Reply:
got it, suggest adding a comment for future maintenance